docker/k8s/mysqlctld/Dockerfile

```diff
@@ -18,3 +18,10 @@ COPY --from=k8s /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificate
 
 # copy vitess config
 COPY --from=k8s /vt/config /vt/config
+
+RUN apt-get update && \
```

Comment:
I would `RUN apt-get` before copying configs. Because of how docker layering works, youll need to run this apt-get every time you change config files. Granted, this is a super minor thing and only really affects build times.